### DISCUSSION TITLE
Categories for file types

### DISCUSSION THREAD

**Discussion by PCUser2021** [setvisible/DownZemAll#208] [Ideas]
Please add categories for files like (Folders like IDM):
Program
Docs
Audio 
Video
Compressed 
General

**Comment by PCUser2021** [comment]
Why does no one respond to my posts after all this time!?

**Comment by setvisible** [comment]
Hi PCUser,

Your ideas are interesting. If you or anyone, wants to implement them, please do. Encouraging self-service reuse is exactly why all the code is open-source.

As is, this software is good enough for my personal own needs. I like it as is. I don't think I'll spend more time on it (except for solving critical bugs).

Regards,
